scopes/react/react/react.env.ts

```diff
@@ -23,7 +23,7 @@ import previewConfigFactory from './webpack/webpack.preview.config';
 import eslintConfig from './eslint/eslintrc';
 
 export const AspectEnvType = 'react';
-const jest = require('jest');
+const jestM = require('jest');
```

Comment:
@ranm8 , any reason to keep this change?

Reply:
Better keep it, seems like the Jest transpiler adds const jest = require('jest') for workspace component testing.